docs/integrations/pytorch_lightning.rst

```diff
@@ -3,719 +3,182 @@
-
-#. |metrics|,
-#. |params|,
-#. |hardware|,
-#. |metadata| including git summary info.
-
-Check this experiment |exp-link| or view quickstart code as a plain Python script on |script|.
-
-|Run on Colab|
-
-.. image:: ../_static/images/integrations/lightning_basic.png
-   :target: ../_static/images/integrations/lightning_basic.png
-   :alt: PyTorchLightning neptune.ai integration
-
-----
-
-.. _advanced-options:
-
-Advanced options
-----------------
-To learn more about advanced options that Neptune logger has to offer, follow :ref:`sections <jump-to-option>` below as each describes one functionality.
-
-If you want to try things out and focus only on the code you can either:
+You just learned how to start logging PyTorch Lightning model training runs to Neptune, by using Neptune logger.
 
-#. Open Colab notebook (badge-link below) and run advanced example as a "`neptuner`" user - zero setup, it just works,
-#. View advanced example code as a plain Python script on |script-advanced|.
+.. warning::
+    This is ``LegacyLogger`` implementation that is under maintenance support only.
+    No new updates will be made to this logger.
 
-You can also check this public project with example experiments: |project|.
+    Please visit
+    `integration docs <https://docs.neptune.ai/integrations-and-supported-tools/model-training/pytorch-lightning>`_
+    to learn about the latest, fully supported version.
 
-|Run on Colab Advanced|
+Use logger inside your lightning Module class
+---------------------------------------------
+You can use log Images, model checkpoints, and other ML metadata from inside your training and evaluation steps.
 
-Before you start
-^^^^^^^^^^^^^^^^
-In addition to the contents of the ":ref:`Before you start <before-you-start-basic>`" section in Quickstart, you also need to have ``scikit-learn`` and ``scikit-plot`` installed.
+To do that you need to:
 
-.. code-block:: bash
-
-    pip install scikit-learn==0.23.2 scikit-plot==0.3.7
-
-Check |scikit-learn| or |scikit-plot| for more info.
-
-.. _jump-to-option:
-
-Jump to your favorite option
-""""""""""""""""""""""""""""
-* :ref:`Advanced NeptuneLogger options <logger-options>`
-* :ref:`Log loss during train, validation and test <Log-loss>`
-* :ref:`Log accuracy score after train, validation and test epoch <log-accuracy-score>`
-* :ref:`Log learning rate changes <log-lr>`
-* :ref:`Log misclassified images for the test set <log-imgs>`
-* :ref:`Log gradient norm <log-grad>`
-* :ref:`Log model checkpoints <log-checkpoints>`
-* :ref:`Log confusion matrix <log-confusion-matrix>`
-* :ref:`Log auxiliary info <log-auxiliary>`
-
-.. _logger-options:
-
-Advanced NeptuneLogger options
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Create ``NeptuneLogger`` with advanced parameters.
+* access the _Run_ object at ``self.logger.experiment``
+* use one of the `Neptune logging methods <https://docs.neptune.ai/you-should-know/logging-metadata#essential-logging-methods>`_
```

Comment:
This should link to legacy logging methods?

Reply:
yep, changed to legacy-docs link